### PR TITLE
[confiscan] Gather OS information

### PR DESCRIPTION
**Pull request by AntonVanAssche**

### CODE REVIEW

File: src/tool/confiscan.sh [hunk 1]
```diff
@@ -115,6 +115,30 @@ else
...
+    "${VERSION_CODENAME}" \
+    "$(uname -r)" >> \
+    "${output_dir}/os_info.csv"
+
+column -s, -t "${output_dir}/os_info.csv"
+
+info "Boot parameters:"
+tr ' ' '\n' < /proc/cmdline | tee -a "${output_dir}/boot_parameters.txt"
+
+column -s, -t "${output_dir}/mount_points.csv"
```
Comment (bdeferme) [inline src/tool/confiscan.sh]: Either I'm blind or I'm missing the part where you're gathering this info :smile:

Comment (AntonVanAssche) [reply src/tool/confiscan.sh]: I think your blind 🙃. All boot params are present in the `/proc/cmdline` file, this is directed to `tr`, which is then appended to the `boot_parameters.txt` file using `tee`.

Comment (bdeferme) [reply src/tool/confiscan.sh]: I was referring to line 140 though, which handles uses mount_points.csv which I guess belongs in PR #18

Comment (AntonVanAssche) [reply src/tool/confiscan.sh]: Whoopsie... 😁

Fixed, see: https://github.com/AntonVanAssche/hogent-bachelorproef/pull/17/commits/8378d6c684752dbbe6e02455c611b2194859d64b.

**Pull request merged**